### PR TITLE
ENG 1619 Create package in npm for

### PR DESCRIPTION
**Pull request by MikeFuster**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fusebit-int/monaco-jsonforms",
```
Comment (bennbollay) [inline package.json:2]: @fusebit

File: package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "version": "1.0.0",
+  "private": true,
```
Comment (bennbollay) [inline package.json:4]: This'll be published publicly.

File: package.json (hunk 1 above)
Comment (MikeFuster) [reply package.json]: Updated

File: package.json (hunk 2 above)
Comment (MikeFuster) [reply package.json]: Set to false

**Review by bennbollay — APPROVED** [review]

**Pull request merged**